### DISCUSSION TITLE
Optimistic or not?

### DISCUSSION THREAD

**Discussion by michal-bartak** [Koenkk/zigbee2mqtt#9276] [General]
What is expected behaviour of optimistic = OFF?

I found, that if light is set to optimistic off, then zigbee2mqtt sends light change to the device, but receives no state back.
I need to refresh its state with refresh button, in Z2M "Exposes" tab.

Is it something expected?

May it be because particular lamp (Hue Go in my example) is not bound to the coordinator at all?
But I cannot bind it any way.

PS TBH I've never had such issues before (HA2021.1 and Z2M 1.18). But recently I updated HA and Z2M (to 1.21.2) and after restarts I found some lights are desynchronized. Then narrowed it down to optimistic feature (still not sure it's the root cause though)

**Comment by michal-bartak** [comment]
Nobody?

**Comment by th3w1zard1** [comment]
Bump as I'm also having this issue. Reporting is correctly turned on for my device.
![image](https://user-images.githubusercontent.com/2219836/229384167-7545abf8-7a86-43da-9cdb-ce72ff357a58.png)

With `optimistic=False`, the state isn't updated in HASS. Using the zigbee2mqtt supervisor addon.
